### PR TITLE
Refactor UI into components

### PR DESCRIPTION
**Pull request by gphorvath**

### CODE REVIEW

**Review by CollectiveUnicorn — CHANGES_REQUESTED** [review]

File: src/lib/themes/index.ts
[the remark this answers is not in the document]
Comment (gphorvath) [reply src/lib/themes/index.ts:1]: Set the default in tailwind.config.js to business/light to address this.

[the remark this answers is not in the document]
Comment (gphorvath) [reply src/lib/themes/index.ts:1]: Changed it to themes.ts

File: src/routes/+layout.server.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /** @type {import('./$types').LayoutServerLoad} */
 export async function load() {
     return {
-        theme: "dark"
+
```
[the remark this answers is not in the document]
Comment (gphorvath) [reply src/routes/+layout.server.ts]: I think this has been resolved

File: src/routes/+page.svelte
[the remark this answers is not in the document]
Comment (gphorvath) [reply src/routes/+page.svelte:1]: Added a panding page

**Review by CollectiveUnicorn — APPROVED** [review]

**Pull request merged**